compiler/justfile

```diff
@@ -0,0 +1,10 @@
+set positional-arguments
+
+init:
+    cargo install cargo-insta
```

Comment:
can we have `cargo-insta` as _dev-dependency_? Because we only need to run `just init` once before `just test` as it depends on `cargo insta`. In _contributing guide_ we don't mention the `just init` command to run. I am unsure where to place `just init` in _contributing guide_. It would be weird to place it just before `just test`.

Reply:
Oh yes, I forgot to include `just init` in the contributing guide. It should probably come just after `cargo install just`

Reply:
Why are we not including it as one of the dependencies?  I didn't get it.

Reply:
Good question. So the dev dependencies are only for libraries. The binaries in those packages are not installed. See this issue: https://github.com/rust-lang/cargo/issues/2267

To get the binaries, we still need to do a `cargo install` which globally installs binaries

Reply:
I see.